acceptance/analyzer_test.go

```diff
@@ -1230,7 +1254,7 @@ func buildAuthRegistryImage(t *testing.T, repoName, context string, registry *ih
 	authConfig, err := auth.BuildEnvVar(authn.DefaultKeychain, regRepoName)
 	h.AssertNil(t, err)
 
-	return regRepoName, authConfig
+	return authConfig
```

Comment:
Ahh I'm sorry, I think I was not clear in my earlier comment. I meant remove `authConfig`, and keep `regRepoName`!

Reply:
Mmm... but  I think we need the **Credentials** created for the registry. right? it is used only once because it is the same registry, but it still a required data to fill the fixture

Reply:
Ohh you're right. The thing is, `auth.BuildEnvVar` - it expects to be called with a repo, but the auth is **per registry**, not per repo. Probably we should refactor `auth.BuildEnvVar` to just expect a registry, but that's another story...

In any case, I think you could call lines 1370+1371 outside of this function with the first repo that is created.